bridge/slack/helpers_test.go

```diff
@@ -0,0 +1,26 @@
+package bslack
+
+import (
+	"testing"
+)
+
+func TestExtractTopicOrPurpose(t *testing.T) {
+	tables := []struct{
+		input string
+		changeType string
+		output string
+	}{
+		{"@someone set channel topic: one liner",    "topic",   "one liner"},
+		{"@someone set channel purpose: one liner",  "purpose", "one liner"},
+		{"@someone set channel topic: multi\nliner", "topic",   "multi\nliner"},
+		{"@someone cleared channel topic",           "topic",   ""},
+		{"some unmatched message",                   "unknown", ""},
+	}
+
+	for _, table := range tables {
```

Comment:
Very nit, please ignore if you feel like it: `tables` / `table` variables could be replaced by `testcases` / `testcase` as it corresponds better to what they represent.